src/dotnet/UI.Blazor/Services/AccountUI/AccountUI.cs

```diff
@@ -61,6 +63,20 @@ public TimeSpan GetPostChangeInvalidationDelay(TimeSpan maxInvalidationDelay)
         return (changedAt + maxInvalidationDelay - CpuClock.Now).Positive();
     }
 
-    public Task SignOut()
-        => ClientAuth.SignOut();
+    public async Task SignIn(string schema)
+    {
+        await ClientAuth.SignIn(schema).ConfigureAwait(false);
```

Comment:
ClientAuth.SignIn is used on ProviderSelectStep. Don't we need to replace it with AccountUI.SignIn?